scripts/lint-golangci-lint.sh

```diff
@@ -17,20 +20,28 @@ else
     exit 1
 fi
 
-echo ">> Linting core"
-cd ${CORE_PATH}
-golangci-lint run ${GOLANG_CI_ARGS}
+if [ ! "${SKIP_NON_GENERATED_FILES}" = true ]; then
+    echo ">> Linting core"
+    cd ${CORE_PATH}
+    golangci-lint run ${GOLANG_CI_ARGS}
+fi
 
 for service_dir in ${SERVICES_PATH}/*; do
     service=$(basename ${service_dir})
     echo ">> Linting service ${service}"
     cd ${service_dir}
-    golangci-lint run ${GOLANG_CI_ARGS}
+    if [ "${SKIP_NON_GENERATED_FILES}" = true ]; then
+        golangci-lint run ${GOLANG_CI_ARGS} --skip-dirs wait
+    else
+        golangci-lint run ${GOLANG_CI_ARGS}
+    fi
```

Comment:
Maybe same here, although here we have the `--skip-dirs` which already gives some more context